### DISCUSSION TITLE
Error GLIBC_2.33 & GLIBC_2.34

### DISCUSSION THREAD

**Discussion by AnSMru** [QuasarApp/CQtDeployer#777] [Q&A] [answered]
У меня установлена Ubuntu 20.04.4 LTS. Установил CQTDeployer версии 1.6.2320.097d82b_Linux.
Но при сборке возникла единственная ошибка о необходимости обновления
/opt/CQtDeployer/bin/CQtDeployer: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.34' not found (required by /opt/CQtDeployer/bin/CQtDeployer)
/opt/CQtDeployer/bin/CQtDeployer: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.33' not found (required by /opt/CQtDeployer/lib/libpe-parse.so)
/opt/CQtDeployer/bin/CQtDeployer: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.33' not found (required by /opt/CQtDeployer/lib/libzip.so)
Но я не могу обновлять версию операционки, необходимо сделать сборку под Ubuntu от 20.04.
Может есть параметры которые мне помогут?

**Comment by EndrII** [comment, marked as answer]
Используйте snap версию cqtdeployer 
она работает на всех дистрибутивах поддерживающих snapd 


``` bash
sudo snap install cqtdeployer
sudo snap connect cqtdeployer:process-control
sudo snap connect cqtdeployer:removable-media
sudo snap connect cqtdeployer:system-backup
```
